README.md

```diff
@@ -503,3 +504,16 @@ total_failures = wavefront_sender.get_failure_count()
   the minor and patch versions.
 * Reach out to us on our public [Slack channel](https://www.wavefront.com/join-public-slack).
 * If you run into any issues, let us know by creating a GitHub issue.
+
+## How to Release
+
+1. Make sure that all changes for the release are merged with the master branch.
+2. Open the `setup.py` file from the top level directory of the project.
+3. Search for `version=` in the file. There you will find the version number for example `1.8.10`.
+4. Log into Github, click on **Releases** in the about section on the right, and click on **Draft a new release.**
+5. For **version**, choose the version you found in step 3, but prefix it with `v` for example `v1.8.10
```

Comment:
For **version**, choose the version you found in step 3, and prefix it with `v` for example `v1.8.101`
 
 The code needs to be closed :) 
 
 Can we add a screenshot here? I tried following the steps but couldn't do it coz am not familiar with the process. A screenshot will help :)